app/Models/Client.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Models;
+
+use Carbon\Carbon;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Client extends Model
+{
+    use HasFactory, SoftDeletes;
+
+    const PAGINATE = 20;
+
+    protected $fillable = ['company', 'vat', 'address'];
+
+    public function getCreatedAtAttribute($created_at)
```

Comment:
I believe overwriting dates fields isn't good. Better add format or something at the end `getCreatedAtFormatted()`

Reply:
It's an accessor used for view purposes only.
Also I can use mutator to save datetime if I send it in different format: https://laravel.com/docs/8.x/eloquent-mutators#defining-an-accessor